pandas/tests/generic/test_metadata.py

```diff
@@ -0,0 +1,58 @@
+import pytest
+
+import pandas as pd
+from pandas.core.meta import PandasMetadata
+
+
+class MyMeta(PandasMetadata):
```

Comment:
Do you have an implementation for index_allows_duplicates?  I'd be more comfortable if there were a more fully fleshed-out example/test.

Reply:
Hmm, I thought I did but am having trouble finding it right now.